Question 1.js

```diff
@@ -0,0 +1,13 @@
+
+let n = prompt("Enter the number");

```

Comment:
`n` is created as a global variable here, but the goal was not to use a global variable.